src/osd/OSD.cc

```diff
@@ -3200,16 +3200,18 @@ int OSD::shutdown()
   cct->get_admin_socket()->unregister_command("ops");
   cct->get_admin_socket()->unregister_command("dump_blocked_ops");
   cct->get_admin_socket()->unregister_command("dump_historic_ops");
-  cct->get_admin_socket()->unregister_command("dump_historic_ops_by_duration");
   cct->get_admin_socket()->unregister_command("dump_historic_slow_ops");
+  cct->get_admin_socket()->unregister_command("dump_historic_ops_by_duration");
```

Comment:
I  am trying to rearrange them  by the order they are registered. So it'll be convenient to find out those singletons next time.
I guess performance is your main concern? That's will be less signicant as we are going to shutdown anyway...